scipy/stats/tests/test_distributions.py

```diff
@@ -7569,3 +7569,73 @@ def test_moment_order_4():
     # has since been made more accurate and efficient, so now this test
     # is expected to pass.
     assert stats.skewnorm._munp(4, 0) == 3.0
+
+
+class TestRelativisticBW:
+    @pytest.mark.parametrize(
+        "rho,gamma", [
+            (36.545206797050334, 2.4952),  # Z0 Boson
+            (38.55107913669065, 2.085),  # W Boson,
+        ]
+    )
+    def test_pdf_integration(self, rho, gamma):
+        """Test pdf integrates to 1."""
+        res = quad(stats.relativistic_bw.pdf, 0, np.inf, args=(rho, 0, gamma))
+        assert_allclose(res[0], 1)
+
+    @pytest.mark.parametrize(
+        "rho,gamma", [
+            (36.545206797050334, 2.4952),  # Z0 Boson
+            (38.55107913669065, 2.085),  # W Boson
+            (96292.3076923077, 0.0013),  # Higgs Boson
+        ]
+    )
+    def test_pdf_against_cdf(self, rho, gamma):
+        """Test whether the integrated PDF matches the CDF."""
+        x = np.arange(0, 100, step=0.01)
+        y_pdf_raw = stats.relativistic_bw.pdf(x, rho, scale=gamma)
+        y_cdf = stats.relativistic_bw.cdf(x, rho, scale=gamma)[1:]
+        y_pdf_cumulative = cumulative_trapezoid(y_pdf_raw, x)
+        assert_allclose(y_pdf_cumulative, y_cdf, rtol=1e-4)
+
+    @pytest.mark.parametrize(
+        "rho,gamma", [
+            (36.545206797050334, 2.4952),  # Z0 Boson
+            (38.55107913669065, 2.085),  # W Boson
+        ]
+    )
+    @pytest.mark.parametrize("n", [1, 2])
+    def test_moments(self, n, rho, gamma):
```

Comment:
Similarly for moments, I think. Please take a look at what's already in `test_continuous_basic.py`.